elastic4s-core/src/main/scala/com/sksamuel/elastic4s/requests/searches/SearchIterator.scala

```diff
@@ -22,7 +22,7 @@ object SearchIterator {
   /**
     * Creates a new Iterator for instances of SearchHit by wrapping the given HTTP client.
     */
-  def hits(client: ElasticClient, searchreq: SearchRequest)(implicit timeout: Duration): Iterator[SearchHit] =
+  def hits[F[_]](client: ElasticClient[Future], searchreq: SearchRequest)(implicit timeout: Duration): Iterator[SearchHit] =
```

Comment:
Thanks, fixed it, it was a typo. I agree that the fact that this code is written in a synchronous way is not the best, however I thought it is better not to touch code like this in this PR, otherwise the scope will be too big. In this case, this code was clearly meant to work with future-based backend implementation, so my idea was to mark it explicitly here. So people who are fine with blocking will use future-based client.